src/javascripts/ingredients/ingredientsData.js

```diff
@@ -4,10 +4,19 @@ import firebaseConfig from '../helpers/apiKeys';
 // API Calls for Ingredients
```

Comment:
I didn't notice this before, but all of your API calls should live within the helper folder.